region/norfair/crocomire/Post Crocomire Farming Room.json

```diff
@@ -545,7 +545,6 @@
       "link": [3, 1],
       "name": "Speedy Jump",
       "requires": [
-        {"notable": "Speedy Jump"},
```

Comment:
moved this to be the tech video (instead of warehouse entrance, since that's just a walljumpless strat)